src/Silk.Core.Logic/Startup.cs

```diff
@@ -1,41 +1,58 @@
-﻿using System.IO;
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
 using System.Threading.Tasks;
+using DSharpPlus;
+using MediatR;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Serilog;
 using Serilog.Events;
+using Serilog.Extensions.Logging;
 using Silk.Core.Data;
 using Silk.Core.Discord;
+using Silk.Core.Discord.EventHandlers;
+using Silk.Core.Discord.EventHandlers.Guilds;
+using Silk.Core.Discord.EventHandlers.MemberAdded;
+using Silk.Core.Discord.EventHandlers.MessageAdded.AutoMod;
+using Silk.Core.Discord.Services;
+using Silk.Core.Discord.Services.Interfaces;
 using Silk.Core.Discord.Utilities;
+using Silk.Core.Discord.Utilities.Bot;
+using Silk.Shared.Abstractions.DSharpPlus.Interfaces;
+using Silk.Shared.Constants;
 
 namespace Silk.Core.Logic
 {
     public class Startup
     {
-        private const string LogFormat = "[{Timestamp:h:mm:ss ff tt}] [{Level:u3}] [{SourceContext:l}] {Message:lj} {Exception:j}{NewLine}";
+        public static DateTime StartupTime { get; } = DateTime.Now;
```

Comment:
I'm pretty sure you can also start replacing `StartupTime` by `DateTime.Now`, unless you got extra plans for this property ^^